tests/ci/data/profiles/rosa-hcp.yaml

```diff
@@ -62,6 +62,34 @@ profiles:
   account-role:
     path: "/test/hcp/"
     permission_boundary: "arn:aws:iam::aws:policy/AdministratorAccess"
+- as: rosa-hcp-ze
```

Comment:
what does this name:rosa-hcp-ze  mean?

Reply:
Why not add zero_egress to exist profile?

Reply:
rosa-hcp-(zero egress)
I added it as a separate profile since there are restrictions about the version itself.